tableauserverclient/models/site_item.py

```diff
@@ -1,8 +1,12 @@
 import xml.etree.ElementTree as ET
-from .property_decorators import property_is_enum, property_is_boolean, property_not_empty, property_not_nullable
+from .property_decorators import (property_is_enum, property_is_boolean, property_matches,
+                                  property_not_empty, property_not_nullable)
 from .. import NAMESPACE
 
 
+VALID_CONTENT_URL_RE = r"^[a-zA-Z0-9_\-]*$"
```

Comment:
Correct, it needs to validate "" or all of our models fail when reading responses from the default site